### PR TITLE
Completed css-brochure-form. - Damien Martinez

### PR DESCRIPTION
**Pull request by damien-martinez**
Screen Shot 2022-01-26 at 9 39 48 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Damien! Your labels and header are not bolded as they are in the example, please revise and submit a new screenshot with your revision.

**Comment by damien-martinez** [comment]
> 
Hey Cody, sorry, completely forgot to add the screenshot.
Screen Shot 2022-01-27 at 2 49 53 PM

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+h3 {
+  text-align: center;
+  color: #ee6c66;
+  text-transform: uppercase;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: 900;
+  font-weight: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:13]: These are only necessary for the `@font-face` rules, you don't need to worry about them. You can just use the font with the `font-family` property.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+label {
+  padding-left: 0.5em;
+  text-transform: uppercase;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:22]: These are only necessary for the `@font-face` rules, you don't need to worry about them. You can just use the font with the `font-family` property. You will need to set the font to be bold though, so you can keep `font-weight`

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  padding-bottom: 0.5em;
+}
+
+.user-input {
+  width: 100%;
+  padding: 0.5em;
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:36]: These are only necessary for the `@font-face` rules, you don't need to worry about them. You can just use the font with the `font-family` property.

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  font-size: 12px;
+  padding: 10px 20px;
+  background-color: #ec5d57;
+  color: white;
+  border-radius: 20px;
+  border: none;
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: bolder;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:57]: These are only necessary for the `@font-face` rules, you don't need to worry about them. You can just use the font with the `font-family` property.

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+.engineer-input {
+  width: 100%;
+  padding: 0.5em;
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:66]: These are only necessary for the `@font-face` rules, you don't need to worry about them. You can just use the font with the `font-family` property.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-brochure-form/styles.css (hunk 2 above)
Comment (damien-martinez) [reply css-brochure-form/styles.css:22]: Hi, thank you for the feedback. I have made the changes.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**